crates/provider/src/docker/client.rs

```diff
@@ -186,21 +186,24 @@ impl DockerClient {
     }
 
     async fn is_using_podman() -> Result<bool> {
-        let result = tokio::process::Command::new("docker")
+        if let Ok(output) = tokio::process::Command::new("docker")
+            .arg("version")
```

Comment:
Using docker alias `docker --version` wasn't returning any string containing podman. Using `docker version` I get an output like this:

```
❯ docker version 
Client:       Podman Engine
Version:      5.0.2
API Version:  5.0.2
Go Version:   go1.22.2
Built:        Tue Jan  1 00:00:00 1980
OS/Arch:      linux/amd64
```